src/config.ts

```diff
@@ -1,10 +1,19 @@
 import { loadConfig, createDefineConfig } from 'c12'
 
+type Color = `#${string}`
+
 export interface DocsConfig {
   name?: string
   description?: string
   github?: string
   redirects?: Record<string, string>
+  /**
+   * The theme color of the documentation site.
+   * It will be used as the `theme-color` meta tag and a full palette of colors will be generated from it.
+   *
+   * @example '#ECDC5A'
+   */
+  theme?: Color
```

Comment:
Maybe `themeColor`? PS: We do support rgb as well as far as I remember.

Reply:
Yes!